fast_matched_filter/fast_matched_filter.py

```diff
@@ -106,12 +125,12 @@ def matched_filter(templates, moveouts, weights, data, step, arch='cpu'):
         for s in range(n_stations):
             for c in range(n_components):
                 #templates[t, s, c, :] -= templates[t, s, c, :].mean()
-                sum_square_templates[t, s, c] = np.sum(
-                    templates[t, s, c, :n_samples_template] ** 2)
+                sum_square_templates[t, s, c] = np.sum(templates[t, s, c, :n_samples_template] ** 2)
```

Comment:
Was there a reason to change this? The stickler in me wants lines less than 80 characters long...